src/lib/auth.js

```diff
@@ -78,3 +77,8 @@ function findUser(profile, accessToken, config, callback)  {
         return callback(false, username + ' is not authorized')
     }
 }
+
+function verifyApiKey(config, req)  {
+    var apiKey = req.headers['apikey'] || req.query['apikey'] || '';
```

Comment:
I would check for the API key in the `Authorization ` header instead of a custom header called `apikey`. Prefix it with `ApiKey` or whatever to look something like `Authorization: ApiKey foobar`.